creator-node/src/components/replicaSet/URSMRegistrationComponentService.js

```diff
@@ -0,0 +1,164 @@
+const axios = require('axios')
+const { promisify } = require('util')
+const crypto = require('crypto')
+const randomBytes = promisify(crypto.randomBytes)
+const { Utils: LibsUtils } = require('@audius/libs')
```

Comment:
i thought i could pull these off the libs instance but couldn't get that working

Reply:
must be due to a static thing